<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Combos</class>
 <widget class="QMainWindow" name="Combos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="comboLayout" stretch="0,0,0,1,0">
      <item>
       <layout class="QHBoxLayout" name="teamLayout" stretch="1,2">
        <item>
         <widget class="QLabel" name="teamLabel">
          <property name="text">
           <string>Team</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="teamCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="titleLayout">
        <item>
         <widget class="QLabel" name="labelCombined">
          <property name="text">
           <string>Combined tasks:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="view"/>
      </item>
      <item>
       <layout class="QGridLayout" name="statsLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="comboLabel">
          <property name="text">
           <string>Combinations</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="pointsLabel">
          <property name="text">
           <string>Bonus points</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="combosEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="pointsEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
